hosting/docker-compose.yaml

```diff
@@ -98,26 +97,15 @@ services:
 
   couchdb-service:
     restart: unless-stopped
-    image: ibmcom/couchdb3
+    image: budibase/couchdb:v3.2.1
```

Comment:
Wondering if we should hard code this - when we release SQS we'll want it to auto upgrade to that version - if we get people to update their docker-compose files sooner then hopefully it should just be an image upgrade. Since this image tag is controlled by us, the latest should always be fine.

Reply:
I did um and ahh about this. Removed. 🙂